### DISCUSSION TITLE
Keep the WINUI Framework Alive: A Plea for Continuity.. Do not abandon WINUI framework!

### DISCUSSION THREAD

**Discussion by nextcodelab** [microsoft/microsoft-ui-xaml#9480] [General]
Dear Microsoft,

I'm writing to express my heartfelt plea: Please do not abandon the WINUI framework! As a dedicated developer, I've recently embarked on the journey of migrating all my apps from UWP to WINUI, and I must say, I'm thoroughly impressed with the capabilities and potential of WINUI.

The transition has been smooth, and the features and flexibility offered by WINUI have truly revitalized my development process. I find myself inspired by the possibilities it presents and the seamless integration it offers across different Windows platforms.

WINUI has become more than just a framework to me; it's a beacon of innovation and a testament to Microsoft's commitment to empowering developers. With its modern design principles and extensive tooling, WINUI has the potential to be the lasting solution for UI development in the Windows ecosystem.

I implore you to recognize the value of WINUI and to continue investing in its growth and evolution. Let's work together to ensure that WINUI remains a cornerstone of UI development, providing developers like myself with the tools and support needed to create exceptional user experiences.

Thank you for your consideration and dedication to advancing the field of software development.

Sincerely

**Comment by AlexanderBlackman** [comment]
This! Winui is already well on the way to being an adequate replacement for WPF/UWP.  
Not everyone needs multiplatform apps.
I hope the VS extension is updated to 1.5, I really want to test a single project WinUI app.

**Comment by chigy** [comment]
@nextcodelab , thank you so much for the love for WinUI! 

Please rest assured, WinUI is not being abandoned. As a matter of fact, the team is working hard to deliver the next release which is part of Windows App SDK 1.6. You can check out the type of features we are working on at https://aka.ms/winappsdk/plans (this page will be updated with the latest release information).
